apps/labs/services/posts/serializePost.ts

```diff
@@ -33,23 +33,24 @@ export const serializePost = async (
     },
   });
 
-  if (!data.author) {
-    throw Error('You did not provide author slug');
-  }
-
-  const postAuthor = authors.find((author) => author.slug === data.author);
-
-  if (!postAuthor) {
-    throw Error('There is not matching author in database');
+  if (!data.authors || data.authors.length == 0) {
+    throw Error('You did not provide any author slug(s)');
   }
 
+  const postAuthors: TPostAuthor[] = data.authors.map((authorName: string) => {
+    const foundAuthor = authors.find((author) => author.slug === authorName);
+    if (!foundAuthor) {
+      throw Error(`Author '${authorName}' is not in the database`);
```

Comment:
This should be a more understandable error message:

```suggestion
      throw Error(`Author '${authorName}' is not defined in Storyblok`);
```